paymentsheet/src/test/java/com/stripe/android/link/ui/signup/SignUpViewModelTest.kt

```diff
@@ -528,6 +532,20 @@ internal class SignUpViewModelTest {
         assertThat(movedToWeb).isFalse()
     }
 
+    @Test
+    fun `isSubmitting state is set while submitting`() = runTest(dispatcher) {
+        val viewModel = createViewModel()
+        val isSubmittingEvents = viewModel.state.map { it.isSubmitting }.distinctUntilChanged()
+        turbineScope {
+            isSubmittingEvents.test {
+                assertThat(awaitItem()).isFalse()
+                viewModel.performValidSignup()
+                assertThat(awaitItem()).isTrue()
```

Comment:
@tillh-stripe re: the other thread -- this assertion would fail without `yield()`.